traversal/selector/fieldKeys.go

```diff
@@ -21,5 +22,6 @@ const (
 	SelectorKey_LimitNone            = "none"
 	SelectorKey_StopAt               = "!"
 	SelectorKey_Condition            = "&"
+	SelectorKey_As                   = "c"
```

Comment:
I realize this would also be a specs change, since we merged that in the `ipld/ipld` repo already, but since this will be the first implementation actually using it --

I still think we should change this key.  There's no way I would either look at the spec and expect it to use the constant "c" here, nor look at the constant "c" in a message and guess remotely correctly what it stands for.  (To wit: I just had to look it up again.)

Can we use "as"?  I don't think a second byte is going to bother anyone, and my surprise factor would go down drastically.

Reply:
i'm very scared to go back to step one given that we're now 2 months in to a very simple change that has now missed 2 shipping deadlines because of the spec then implementation then wait long time for review approach.

Reply:
https://github.com/ipld/ipld/pull/170